### ISSUE TITLE
ON HOLD - Date picker is not fully accessible

### ISSUE THREAD

**Issue by kuzmenkn** [CDH-Studio/UpSkill#1705]
Accessibility notes:	
1. When the date picker opens by clicking the ‘Choose date’ button and a date in the calendar grid receives focus, the screen readers should announce keyboard instructions for navigating the calendar grid. The instructions could be visible at the bottom of the dialog box.
2. When the month or year of the calendar grid changes as users navigate the calendar or activate the buttons for the next or previous month or year, the screen readers should announce the new month and year.
3. When a date is chosen, the screen reader should announce the selected date. So, when the dialog closes and the focus returns to the Choose Date button, screen reader users hear confirmation of the selected date in the button name.

![image](https://user-images.githubusercontent.com/90463917/139477555-d7e30011-4854-4f35-a175-a42c13515285.png)

**Comment by MenaMachado** [comment]
Hi Rizvi, the field allows users to type in the date, but either replace "Select date" in field to EN (YYYY-MM) FR (AAAA-MM) or add it to end date (ie End date (YYYY-MM))

EN
![image](https://user-images.githubusercontent.com/49656364/143616411-fd8440ad-c41d-4612-96ee-224bb3fd6f4b.png)

FR
![image](https://user-images.githubusercontent.com/49656364/143616991-0199a172-b7aa-40de-8f2e-fdd94b753ff1.png)
